unimport/__main__.py

```diff
@@ -63,6 +70,19 @@ def show(
         )
 
 
+def get_gitignore(root: Union[Path, str]) -> PathSpec:
+    """Return a PathSpec matching gitignore content."""
+    if not PathSpec:
+        return None
+    lines: List[str] = []
```

Comment:
```suggestion
```

Reply:
This line shouldn't be removed, because `.gitignore` may not exist!